docs/design/specs/Ecma-335-Augments.md

```diff
@@ -745,6 +745,10 @@ T Exponential<T, TFloat>(T exponent) where
 Another example of polymorphic maths calculating the exponential using the Taylor series,
 usable for calculating the exponential of a matrix.
 
+### II.14.3 Enums (page 168)
+
+Enum support for bool as underlying type was deleted (see [issue 79224](https://github.com/dotnet/runtime/issues/79224)).
```

Comment:
This change is deleting the bool backed enum support from the reflection stack. 

The bool backed enums are still going to work loaded fine by the type loader. I think that it is the right tradeoff between compat and simplicity. The type loader support for bool backed enums is very cheap and simple.

ECMA-335 and this doc is concerned about the behavior of the core runtime (that is not changing), not about behavior of BCL APIs. I think this note should be deleted.

Reply:
What is the 'type loader' ?

Reply:
Type loader is a component responsible for loading types. 

For example, `Console.WriteLine(typeof(BoolEnum));` is still going to work fine even with these changes because the type loader still works fine for bool enums with these changes.